Certification-Standard/introduction.md

```diff
@@ -0,0 +1,239 @@
+# Cardano Certification Standard
+
+**Draft version**
+**Last modification September 21th 2023**
+**Authors:**
+
+- Romain Soulat
+
+## 1 Introduction
+
+The Cardano Certification Standard (CCS) is a framework designed to facilitate the comparability of independent evaluations within the Cardano blockchain ecosystem. This standard establishes a comprehensive set of requirements for assessing the security features of Decentralized Applications (DApps) on the Cardano blockchain, as well as the assurance measures applied during security evaluations.
+
+The primary objective of the evaluation process is to establish a level of confidence that the security functionalities of these DApps and the assurance measures align with the predefined requirements. Consequently, the evaluation results serve as a valuable resource for end-users seeking to gauge whether a particular DApp aligns with their security requirements and expectations.
+
+The Cardano Certification Standard also serves as a guide for both DApp developers and evaluators. It offers insights and best practices for the development of DApps with robust security features and provides a structured approach for evaluating their security posture.
+
+Flexibility is a core principle of the Cardano Certification Standard. It is designed to accommodate a diverse range of evaluation methodologies, ensuring that all essential security properties required by a DApp can be effectively assessed. However, it is essential to note that the validity of an evaluation is contingent upon the specific security properties evaluated and the methods employed. Auditors and users are advised to examine the DApp's security requirements and evaluation methodologies to determine the meaningfulness of the evaluation results.
+
+The Cardano Certification Standard aims at adressing:
+
+- The protection of assets of any user interacting with the DApp
+- The loss of use of the DApp
+- ...
+
+Moreover, the scope of the Cardano Certification Standard could extend to security aspects beyond these categories.
+
+Nevertheless, it is important to acknowledge that certain topics are explicitly excluded from the purview of the Cardano Certification Standard.
+
+Among these exclusions are:
+- ...
+
+In conclusion, the Cardano Certification Standard serves as a document promoting transparency, consistency, and confidence in the security of DApps.
+
+*RSoulat* This document draws heavily from the work of Common Criteria. I think we should take some inspiration from it for the security aspects, and, if we want to deal with purely functional aspects, add them where we see fit.
```

Comment:
General comment that I may as well make here.

Given the dependency on the common criteria, I wondered whether that should be more explicit. That could be done in two ways.

1. By including cross references to the common criteria standards in appropriate places in this document.
2. By writing a different kind of document that is effectively a diff with the common criteria document.

The point about doing this is that it makes comparison with the existing standard much easier, and also allows evolution of this standard as it grows and also evolution with the common criteria standard as it evolves.

Reply:
My issue with doing a diff is that the CC are a very large set of documents for the security of IT products. Some of the concepts, I believe don't need to be applied to our situation and I'm not sure it would make sense to have a diff. 
I agree we could cross-reference though.